docs/FrontierDevelopments-CAPI-endpoints.md

```diff
@@ -14,138 +16,373 @@ It's safest to store the token_type that comes back from the request for
+1. `name` - Station name
+1. `outpostType` - What type of outpost it is, examples: `starport`, `fleetcarrier`
+1. `imported` - What commodities that are imported from this station (dictionary, commodity id and key)
+   - Example `"128049162": "Cobalt"`
+1. `exported` - What commodities that are exported from this station (dictionary, commodity id and key)
+   - Example `"128049162": "Cobalt"`
+1. `services` - What services the station provides and their status
+	1. `dock` - Status of Dock (`ok`, `unavailable`, `private`)
+	1. `contacts` - Status of Contacts (`ok`, `unavailable`, `private`)
+	1. `exploration` - Status of Exploration (`ok`, `unavailable`, `private`)
+	1. `commodities` - Status of Commodities (`ok`, `unavailable`, `private`)
+	1. `refuel` - Status of Refuel (`ok`, `unavailable`, `private`)
+	1. `repair` - Status of Repair (`ok`, `unavailable`, `private`)
+	1. `rearm` - Status of Rearm (`ok`, `unavailable`, `private`)
+	1. `outfitting` - Status of Outfitting (`ok`, `unavailable`, `private`)
+	1. `shipyard` - Status of Shipyard (`ok`, `unavailable`, `private`)
+	1. `crewlounge` - Status of Crewlounge (`ok`, `unavailable`, `private`)
+	1. `searchrescue` - Status of Search & Rescue (`ok`, `unavailable`, `private`)
+	1. `techbroker` - Status of Tech broker (`ok`, `unavailable`, `private`)
+	1. `stationmenu` - Status of Station menu (`ok`, `unavailable`, `private`)
+	1. `shop` - Status of shop (`ok`, `unavailable`, `private`)
+	1. `engineer` - Status of engineers (`ok`, `unavailable`, `private`)
+1. `economies` - What different types of economies you can expect from the station (dictionary, economyId as key)
+	- Example `"23": { "name" : "HighTech", "proportion": 0.7 }`
+1. `prohibited` - What commidities that are prohibited on this station (dictionary, commodityId as key)
+   - Example `"128049162": "Cobalt"`
+1. `commodities` - All available commodities and their info
+	1. `id` - Commodity ID
+	1. `name` - Commodity name
+	1. `legality` - Status of the legality of this commodity at this station, empty string = legal
+	1. `buyPrice` - The buy price at the moment of the visit
+	1. `sellPrice` - The sell price at the moment of the visit
+	1. `meanPrice` - The mean price at the momenf of the visit
+	1. `demandBracket` - How much in demand this commodity is, value between `0` and `3`
+	1. `stockBracket` - How much in stock this commodity has (bracket), value between `0` and `3`
+	1. `stock` - How many items are in stock of this commodity
+	1. `demand` - How many items the station has demand for (more is better)
+	1. `statusFlags`
+	1. `categoryName` - What type of category the commodity belongs to
+	1. `locName` - Localised name
+
+&nbsp;
+
+---
+
 ## Journal
 
+Gives access to the authenticated players journals, all sessions are combined into a single file for a single request
+
+### Request endpoints
+
+-   `GET <https://companion.orerve.net/journal>`
+-   `GET <https://companion.orerve.net/journal/[year]/[month]/[day]>`
```

Comment:
specify it's a 4-digit year (I've never tried with 2-digit myself, and suspect my own code won't let me).

Reply:
Yeah, sure. Fair enough. ^^